File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/examples/simple/SinReciprocalZ.java
package net.fwitz.math.main.binary.complex.functions.examples.simple;

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class SinReciprocalZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("sin(1/z)")
                .computeFn(z -> z.inverse().sin())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/examples/simple/SinReciprocalZ.kt
package net.fwitz.math.main.binary.complex.functions.examples.simple

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object SinReciprocalZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("sin(1/z)")
        .computeFn { z -> z.inverse.sin }
        .render()
}